src/IBM.WatsonDeveloperCloud/Http/Request.cs

```diff
@@ -109,6 +109,7 @@ public TaskAwaiter<IResponse> GetAwaiter()
         public async Task<T> As<T>()
         {
             HttpResponseMessage message = await this.AsMessage().ConfigureAwait(false);
+            var result = message.Content.ReadAsStringAsync().Result;

```

Comment:
@mediumTaj Who has the Request is the WatsonHttpClient, and therefore the Visual Recognition abstraction has the dependency of the WatsonHttpClient. In order for the client to have access to the request, WatsonHttpClient needs to provide this.